computing-resources/neuropoly/data/git-datasets.md

```diff
@@ -347,7 +354,11 @@ cat id_rsa.zamboni.pub | ssh git@data.neuro.polymtl.ca keys add zamboni
 You can also paste the key in, followed by `ctrl-d`; this looks like:
 
 ```
-$ ssh git@data.neuro.polymtl.ca keys add zamboni
+ssh git@data.neuro.polymtl.ca keys add zamboni
+```
+
+The output looks like:
+```
```

Comment:
This was also on purpose, re: https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/31#pullrequestreview-821055047